snippets/base/static/js/admin/sms-country-toggle.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+    function bindSMSCountry() {
+        const chk_sms = document.querySelector('.form-row.field-include_sms input');
+        const country_row = document.querySelector('.form-row.field-country');
+        const country_input = document.querySelector('.form-row.field-country input');
+
+        // make sure required fields exist
+        if (chk_sms && country_input) {
+            const country_default = country_input.value;
+            
+            function displayCountryField() {
+                country_row.classList.add('visible');
+            }
+
+            function hideCountryField() {
+                country_row.classList.remove('visible');
+                
+                // set the value back to default if user left it empty
```

Comment:
This doesn't seem to work, field remains empty and raises an error when saving

Reply:
hm, can you paste the error you're getting? it's working as expected on my end.

Reply:
I get a generic "Please correct the error bellow" with no errors displayed, b/c the field is hidden. If I check the SMS option, I see the field and it's empty and the error is "This field is required".

STR:
 1. Create a new S2D snippet
 2. Fill in all required fields
 3. Make sure to check include sms
 4. Empty country field
 5. Save
 6. Snippet does not save, correctly, due to empty country field
 7. Uncheck `include sms`
 8. Save
 9. Snippet still doesn't save, incorrectly this time b/c country field is not visible and should default to `us`.

Reply:
hm, i believe i have this fixed. however, i cannot create a snippet, as the first `Locale` drop down has no options (and is required).

how can i populate that select box? i've run the commands in the readme (`create_locales()`) and see locale data in the db (`TargetedLocale.objects.all()`).

Reply:
i figured it out while working on #996. 😄 followed the STR above and was able to successfully save the snippet.